teams/team-gitops/dev/cluster1/infra/provider-config-aws-amd-1-29-blueprint.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: aws.upbound.io/v1beta1
```

Comment:
Why are we duplicating this file? Can we not have one common upbound provider config for all clusters?